md-docs/PM4ML_Docs/TestTransfer.md

```diff
@@ -0,0 +1,87 @@
+# Test Transfer Process
+
+This guide outlines the process for sending test transfers to verify connectivity across three transfer types: Outbound Transfers, Inbound Transfers, and FXP Transfers. The steps include how to initiate transfers, check results, and troubleshoot failures using the Testing Toolkit (TTK).
+
+---

```

Comment:
I think its worth mentioning that this testing process is for local development and not for testing transfers when connected to live mojaloop switch. The process may vary when connected to mojaloop switch.